### PR TITLE
Improve FileDrop View

### PR DESCRIPTION
**Pull request by hurradieweltgehtunter**
## Description
Improve Layout of FileDrop view

## Motivation and Context
The FileDrop View is an external endpoint, which gets presented to users possible not knowing what a FileDrop or even ownCloud is. The current layout does not provide any information about it and does not offer clear instructions what to do, where to click.
Also, current status of (german) translations are ... well ... 💩 

## Screenshots (if appropriate):

Before:
grafik

After:
grafik

Drag `n Drop state:
grafik

After upload showing file infos:
grafik

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:
- [x] Clarify why code changes are not compiled 
- [ ] Add translation strings to transifex and back

### CODE REVIEW

**Comment by hurradieweltgehtunter** [comment]
Not sure what I'm doing wrong, but while changes to other components or views get compiled and shown in browser immediately, the changes in the filedrop view do not have any impact in the browser 🤷 Caching issue maybe?

**Comment by kulmann** [comment]
> Not sure what I'm doing wrong, but while changes to other components or views get compiled and shown in browser immediately, the changes in the filedrop view (see changed files) do not have any impact in the browser 🤷 Caching issue maybe? (is there even a cache?)
> Docker + Vite is running

Ok so that's a little mean... the backend doesn't know anything about you running the frontend with vite. So the public link that you copy is on port 9200, but vite is running on port 9201. So for public links you need to modify the copied URL to get the pleasure of the vite dev experience.

Example:
copied url: `https://host.docker.internal:9200/s/ZbBsOLkHqannLUt`
manually modified url: `https://host.docker.internal:9201/index.html#/s/ZbBsOLkHqannLUt`

**Comment by hurradieweltgehtunter** [comment]
Before:
grafik


After:
grafik

**Comment by hurradieweltgehtunter** [comment]
Some issues I was not able to solve:
- How to hide the default snackbar/notification upload status bottom right? Would be a duplicate 
- How to get the new strings into transifex and back?

File: packages/web-app-files/src/views/FilesDrop.vue [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
+            <h2>
+              <span v-text="$gettext('An error occurred while loading the public link')" />
+            </h2>
+            <p class="oc-m-rm" v-text="errorMessage" />
+          </div>
+
+          <div class="explanation oc-flex oc-flex-center oc-mt-l">
+              <div class="oc-width-1-2@m oc-width-1-3@xl oc-pt-l">
+                <h2 class="oc-text-center" v-text="$gettext('What is this?')" />
+                <p v-text="$gettext('You can upload files here simply by drag `n drop of click on “Choose a file“ to open a file selection box.')" />
```
Comment (C0rby) [inline packages/web-app-files/src/views/FilesDrop.vue:45]: A small typo.
```suggestion
                <p v-text="$gettext('You can upload files here simply by drag `n drop or click on “Choose a file“ to open a file selection box.')" />
```

**Comment by ChrisEdS** [comment]
image

Can you try to make that line a bit more prominent? Also, I'm not sure what the original string is, but we should remove "Folder" and add "Resource". Technically this is a folder, but user facing there is no "folder" - it's a bit misleading.

**Comment by hurradieweltgehtunter** [comment]
> image
> 
> Can you try to make that line a bit more prominent? Also, I'm not sure what the original string is, but we should remove "Folder" and add "Resource". Technically this is a folder, but user facing there is no "folder" - it's a bit misleading.

I made it “un”-prominent on purpose because it does not contain any *primary* relevant information to the receiving user. The only *primary* information IMO is “Choose a file or drag it here”, that's the primary action and the whole purpose of this page. Everything else is subordinate to this.

folder/resource: Yes, you are right, I'm not happy with the current state. But since we'll work on the textual contents next week, I'd wait for the outcome, then see what fits best.

**Comment by ChrisEdS** [comment]
> I made it “un”-prominent on purpose because it does not contain any primary relevant information to the receiving user. 

The information who invited me to upload files seems important to me, but I'm good with the current look as well! 

This enhancement will go into 7.0.0, right? Is there something I can help to speed up things?

**Comment by hurradieweltgehtunter** [comment]
> > I made it “un”-prominent on purpose because it does not contain any primary relevant information to the receiving user.
> 
> The information who invited me to upload files seems important to me, but I'm good with the current look as well!
> 
> This enhancement will go into 7.0.0, right? Is there something I can help with to speed up things?

Some work is in the making. Some more patience required :)

**Comment by hurradieweltgehtunter** [comment]
Final layout after talking to marketing and @tbsbdr  looks like this:

grafik

**Comment by ChrisEdS** [comment]
Love it!

**Comment by hurradieweltgehtunter** [comment]
Done from my side. Left to do: handle translations (help needed)

**Comment by ChrisEdS** [comment]
Yes, and it is way to big and can be moved more to the bottom?

**Comment by hurradieweltgehtunter** [comment]
> Love it so far but the text "You can drag n drop the file here or choose to upload" is a little unnecessary in my opinion.
> 

Do you mean the "What is this" text part? IMO a short explanation of what the user sees and is able to do might be essential. The receiving user might not know, at all, what to do, especially since he or she is for sure not registered on the serving instance.

**Comment by JanAckermann** [comment]
IHMO choose a file or drag it here is self explaining. I don't think we add much value telling the user, that a file picker will open.

**Comment by tbsbdr** [comment]
I'd love to discuss and improve this at the beginning of may

**Comment by ChrisEdS** [comment]
@tbsbdr Can we continue with that PR?

**Comment by tbsbdr** [comment]
## Initial
- Info: `Albert Einstein is requesting files`
- Instruction: `Drop files here to upload, or use the 'Upload' button.`
- CTA button in the center of attention `Upload` (same as in files app)
- more decent note "Existing content is not revealed. Only Albert Einstein can see  uploads."
 
image

## Success
- Info: `Albert Einstein is requesting files`
- Success confirmation: ✅ `Upload finished`
- Scrollbox lists sucessfully uploaded files
- start over option: `Add more files` (user stays on site and sees always which files she had already uploaded - list grows)
- summary `Total 1 file, 4.4 MB`

image

**Comment by kulmann** [comment]
Thanks a ton for driving this @hurradieweltgehtunter ! To get it finally done, this is now superseded by #10438

**Pull request closed without merging**